insights/client/client.py

```diff
@@ -565,6 +567,30 @@ def handle_startup():
     if config['validate']:
         return validate_remove_file()
 
+    # handle cron stuff
+    if config['enable_schedule'] is True and config['disable_schedule'] is True:
+        logger.error('Conflicting options: --enable-schedule and --disable-schedule')
+        return False
+
+    if config['enable_schedule']:
+        # enable automatic scheduling
+        InsightsSchedule()
```

Comment:
It seems more intuitive to me to not call `set_daily` in `InisghtsSchedule.__init__` and instead call it here.  What do you think?

Reply:
Done.